### PR TITLE
Fix reboot problem

### PR DESCRIPTION
**Pull request by Bischoff**
## What does this PR change?

* regenerate kernel on SLE 11 SP4
* style and English fixes

## Links

See also uyuni-project/sumaform#1018

Ports:
* 4.1: SUSE/spacewalk#16458
* 4.2: SUSE/spacewalk#16457

## Changelogs

- [x] No changelog needed

### CODE REVIEW

File: testsuite/features/step_definitions/command_steps.rb [hunk 1]
```diff
@@ -1717,22 +1717,30 @@
...
+  raise "Folder exists" if folder_exists?($server, folder)
 end
-When(/^I ensure folder "(.*?)" doesn't exists$/) do |folder|
-    folder_delete($server, folder) if folder_exists?($server, folder)
+
+When(/^I ensure folder "(.*?)" doesn't exist$/) do |folder|
+  folder_delete($server, folder) if folder_exists?($server, folder)
+end
+
+When(/^I regenerate the boot RAM disk on "([^"]*)" if necessary$/) do |host|
```
Comment (ktsamis) [inline testsuite/features/step_definitions/command_steps.rb:1739]: I would put here instead "I regenerate the boot RAM disk on "([^"]*)" if its a SLES 11" to make sure that the check is not done in kernel files or anything like that but we check the OS

**Review by ktsamis — CHANGES_REQUESTED** [review]
This is the only thing that is cosmetic pretty much. I don't mind the current name but mayyyybe it's better to be explicit that this only happens on 11

File: testsuite/features/step_definitions/command_steps.rb (hunk 1 above)
Comment (Bischoff) [reply testsuite/features/step_definitions/command_steps.rb:1739]: The problem is, we don't know if it's not necessary on other systems.

It just happens that a kernel patch is the first in the list on SLES 11. That's why we know the problem only on SLES 11 so far.

 Let's imagine it's needed on SLE 12 as well. I prefer to change only on implementation than in implementation and title.
I prefer the step name to remain conceptual and leave the details to the implementation.

Comment (ktsamis) [reply testsuite/features/step_definitions/command_steps.rb:1739]: Ok then let's leave it as is and we change the implementation if needed later on. All good then

**Review by ktsamis — APPROVED** [review]

**Pull request merged**